deflect/StreamPrivate.cpp

```diff
@@ -117,7 +117,9 @@ bool StreamPrivate::finishFrame()
 {
     // Open a window for the PixelStream
     const MessageHeader mh( MESSAGE_TYPE_PIXELSTREAM_FINISH_FRAME, 0, name );
-    return socket.send( mh, QByteArray( ));
+    const bool ret = socket.send( mh, QByteArray( ));
```

Comment:
I have done tests myself on this one today and I think it has nothing to do with the socket. Somehow I have noticed that the opened windows in DC are sometimes getting the tap in second time. May be related.